### ISSUE TITLE
Staging Review finding: Error message not announced to screen reader users

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#91815]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Management Tools
**Product:** Claim Status Tool
**Feature:** Improved 5103 requests and waiver experience

## Findings details
**VA.gov Experience Standard - issue:** [User doesn't have enough information to complete a task.](https://design.va.gov/about/experience-standards/#user-has-enough-information-to-complete-a-task)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** Yes
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On the 5103 Evidence Notice page, if you press "Submit evidence waiver" without checking the checkbox, you get an inline error message, "You must confirm you’re done adding evidence before submitting the evidence waiver." However, that message is not announced to screen reader users when it appears.

Without the visual cue of the new text added to the screen, you're not likely to realize what's happened. Did the waiver submit with no announcement? Was there an error? Is the page just broken? etc. Screen reader users have to realize that it didn't work and then explore the page to figure out why it didn't work.

<details><summary>Link, screenshot or steps to recreate</summary>

Steps to reproduce:
1. Log in with test user Kyle Cole.
2. Turn on a screen reader (easiest for testing: VoiceOver on Mac or NVDA on Windows).
3. Tab to the "Submit evidence waiver" button. (Don't check the checkbox.)
4. Press [enter].
5. Observe that no announcement is made when the error message appears.

</details>

## Recommended action

In the forms library when there's an inline error, we send focus to the input that has the error message. That would be ideal behavior here. Moving focus ensures that the input and the error message are immediately announced.

## References

- **Accessibility Defect Severity:** [1: Critical. Must be fixed before launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-1)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by davidakennedy** [comment]
I meant with @briandeconinck yesterday after our staging review. I showed him this error properly announcing on our call. In addition, I recorded it working in a video. Brian said we can close this as working as expected. @jacobworrell Can you do the honors of closing?

https://github.com/user-attachments/assets/fc3bbc34-e7b5-45d0-a5fa-199dacdce701
